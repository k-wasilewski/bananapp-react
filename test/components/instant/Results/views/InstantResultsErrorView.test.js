import React from 'react';
import renderer from 'react-test-renderer';
import store from "../../../../../src/redux/store";
import {BrowserRouter} from "react-router-dom";
import {Provider} from "react-redux";
import InstantResultsErrorView from "../../../../../src/components/instant/Results/views/InstantResultsErrorView";

describe("InstantResultsErrorView rendering specification", () => {
    it('InstantResultsErrorView is rendered', () => {
        const component = renderer.create(
            <Provider store={store}>
                <BrowserRouter>
                    <InstantResultsErrorView />
                </BrowserRouter>
            </Provider>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});