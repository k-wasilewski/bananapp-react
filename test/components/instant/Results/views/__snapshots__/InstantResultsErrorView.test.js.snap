// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstantResultsErrorView rendering specification InstantResultsErrorView is rendered 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h3>
      File type not supported or it's not a banana, try again
    </h3>
    <a
      href="/"
      onClick={[Function]}
    >
      <button
        variant="outlined"
      >
        Back
      </button>
    </a>
  </header>
</div>
`;
